From 099ff9994fbfec805887d878d1dec7cd28ffbfad Mon Sep 17 00:00:00 2001
From: Bo Huang <bo.b.huang@intel.com>
Date: Wed, 16 Oct 2013 14:18:16 +0800
Subject: [PDK KK AOSP]: Resolve dependency issue of ADB.

BZ: 144066

KK pdk change dependency and impact on adb build.

Change-Id: Ic6b3eaf5b646a5ca781e97d1270d8dc40ea93064
Category: aosp improvement
Domain: USB ADB
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: Bo Huang <bo.b.huang@intel.com>
---
 adb/sockets.c |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/adb/sockets.c b/adb/sockets.c
index 5581cd7..d79bea3 100644
--- a/adb/sockets.c
+++ b/adb/sockets.c
@@ -23,6 +23,10 @@
 
 #include "sysdeps.h"
 
+#if !ADB_HOST
+#include <cutils/properties.h>
+#endif
+
 #define  TRACE_TAG  TRACE_SOCKETS
 #include "adb.h"
 
-- 
1.7.4.1

